problem_builder/tests/integration/test_step_builder.py

```diff
@@ -765,3 +774,14 @@ def test_plot(self):
             step_builder, plot_controls, hidden=False,
             labels=['Custom Q1 label', 'Custom Q2 label', 'Custom Q3 label', 'Custom Q4 label']
         )
+
+    def test_instruction_message(self):
+        step_builder, controls = self.load_assessment_scenario("step_builder_instruction.xml", {})
+        # Step 1
+        # Submit free-form answer, go to next step
+        self.freeform_answer(None, step_builder, controls, 'This is the answer', CORRECT, hold=True, last=True)
+        message = step_builder.find_element_by_css_selector('.sb-step-message')
+        self.wait_until_visible(message)
+        self.assertEqual(message.text, 'Hello!')
+        controls.review.click()
```

Comment:
@Kelketek It might make sense to extend the test scenario to include two steps: That would allow to not only check if the message disappears when clicking one of the controls for navigating between steps (as you are doing here), but also when clicking anywhere, e.g. via something like

```js
self.browser.execute_script("$(document).trigger('click')")
```

The reason I bring this up is because both "Next" and "Review grade" implicitly call `hideAllSteps` when clicked, which also has the effect of hiding the message (at least until you come back to the same step).